pkg/network/node/pod.go

```diff
@@ -79,6 +86,8 @@ func newPodManager(kClient kubernetes.Interface, policy osdnPolicy, overlayMTU u
 	pm.routableMTU = routableMTU
 	pm.podHandler = pm
 	pm.ovs = ovs
+	pm.podInformers = kinformers.NewSharedInformerFactory(kClient, defaultPodInformerResyncPeriod)
```

Comment:
This informer is just used for pods on this node, right? So you can actually use a FieldSelector to limit the pods that we watch here. That will save a lot of memory.

Here's an example for how to do this: https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/factory/factory.go#L259